lifecycle-image/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Stage 1
+FROM ubuntu:bionic AS download-and-extract-lifecycle
+
+ARG LIFECYCLE_VERSION
+ARG LIFECYCLE_GITHUB_ORG
```

Comment:
Can we give this a default? Or better yet just hardcode `buildpacks` in the `LIFECYCLE_URI` default. I imagine folks will typically override the entire `LIFECYCLE_URI` if they want to change the artifact location.

Reply:
Yeah, this is an outgrowth of using my fork for testing... I will make the change.